dm/relay/remote_reader.go

```diff
@@ -74,8 +74,8 @@ type reader struct {
 }
 
 // NewReader creates a Reader instance.
-func NewReader(cfg *Config) Reader {
-	return &reader{
+func NewReader(cfg *RConfig) Reader {
```

Comment:
now we have relay.NewBinlogReader and relay.NewReader, need a better name.

Or in futre we can wrap relay.NewBinlogReader into another public method like relay.StartPulling?

Reply:
will be that way in next pr.